hudi-spark-datasource/hudi-spark-common/src/main/scala/org/apache/spark/sql/execution/datasources/parquet/HoodieFileGroupReaderBasedParquetFileFormat.scala

```diff
@@ -65,6 +66,8 @@ class HoodieFileGroupReaderBasedParquetFileFormat(tableState: HoodieTableState,
                                                   requiredFilters: Seq[Filter]
                                            ) extends ParquetFileFormat with SparkAdapterSupport with HoodieFormatTrait {
 
+  override def getRequiredFilters: Seq[Filter] = requiredFilters
```

Comment:
Do we already support `hoodie.data source.read.incr.filters` for incremental pulls in the new Hudi parquet file format?

Reply:
No. The override is implementing the method from the trait. I think "override" might not be necessary

Reply:
I'm not particularly asking about this line.  We support `hoodie.data source.read.incr.filters` in `IncrementalRelation`.  Wondering if we also have the same support in the new file format class.